foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClientException.java

```diff
@@ -5,8 +5,23 @@
 
 /** Generic exception for errors occurring when using OpenAI foundation models. */
 public class OpenAiClientException extends RuntimeException {
+  static final String BASE_ERROR_MESSAGE = "Request to OpenAI model failed";
   @Serial private static final long serialVersionUID = -7345541120979974432L;
 
+  /** Create a new exception with the base message: {@code Request to OpenAI model failed} */
+  public OpenAiClientException() {
```

Comment:
(Major/Discussion)

Please don't allow for lazy shortcuts. Every instance where exceptions are thrown, it should have a distinct error message (and cause) such that troubleshooting remains easy.

Reply:
I removed it